confdb/confdb_test.go

```diff
@@ -3174,7 +3191,10 @@ func (*viewSuite) TestDetectViewRulesExpectDifferentTypes(c *C) {
 	err = view.Set(bag, "a[0]", "foo")
 	c.Assert(err, ErrorMatches, `key "\[0\]" cannot be used to access map at path "a\[0\]"`)
 
-	err = view.Unset(bag, "a")
+	err = view.Unset(bag, "a[0]")
+	c.Assert(err, ErrorMatches, `key "\[0\]" cannot be used to access map at path "a\[0\]"`)
```

Comment:
isn't the map at path "a" ?

Reply:
I've change this (and all other strings with the same format) to be a bit clearer